components/Layout/HomeHeader/SwiperProgressBar.vue

```diff
@@ -62,6 +62,13 @@ export default Vue.extend({
       window.requestAnimationFrame(this.updateBars);
     });
   },
+  destroyed() {
+    const animEndFunction = this.onAnimationEnd;
+
+    this.bars.forEach(function (el: HTMLElement) {
```

Comment:
Any reason for not using an arrow function?

Reply:
@MrTimscampi None really, never used them myself. Changed all the occurences in the code of this component, thanks for the suggestion! 👍